<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createQuestion</class>
 <widget class="QWidget" name="createQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="questionNumber">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Question X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>560</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>621</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="C">
        <property name="text">
         <string>Answer C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="answerC"/>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonC">
        <property name="text">
         <string>             </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="B">
        <property name="text">
         <string>Answer B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="answerB"/>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonB">
        <property name="text">
         <string>             </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="labelQuestion">
        <property name="text">
         <string>Question</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="Question"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="A">
        <property name="text">
         <string>Answer A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="answerA"/>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonA">
        <property name="text">
         <string>             </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="D">
        <property name="text">
         <string>Answer D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="answerD"/>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonD">
        <property name="text">
         <string>             </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="correctAnswer">
      <property name="text">
       <string>Correct answer:  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
